### ISSUE TITLE
Inline tooltip is too aggressive

### ISSUE THREAD

**Issue by Nopik** [sourcegraph/jetbrains#1871]
### IDE Information

Latest Idea

### Bug Description

When editing text in IDE, trying to select almost full line is now prohibited by Cody aggressive tooltip. In some cases I want to select text from current cursor to the end of line, except last few characters. So I press Shift+Cmd+Right (not sure if this is default keystroke), then I typically press Left few times keeping Shift to maintain selection. But with the latest Cody update (6.0.12) it is impossible, cursor is stuck after the Cody tooltip and does not move left anymore.

<img width="288" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/554841/5388c654-5f16-4a6e-91af-ba0d0a879ef6">

**Comment by Nopik** [comment]
As a matter of fact, that tooltip blinks hundred times per hour, when editing text. I'm finding it extremely annoying, would be great if it can be disabled completely, or at least delayed by a second or two.

**Comment by mdhender** [comment]
I stopped using the nightly builds for JetBrain because of this. Looks like I have to figure out how to downgrade the plug-in or uninstall it until there's a way to disable this feature.

**Comment by Nopik** [comment]
Too bad, that this feature was released together with Claude 3.5 support, so by downgrading you lose the model as well. Otherwise I'd downgrade, too.

**Comment by Nopik** [comment]
Please, let users disable this tooltip. Please. It is extremely annoying. Every single time I do text selection, which often happens several time per minute, it shows up. And every single keystroke to extend the selection, the tooltip jumps over the screen. Every. Single. Keystroke. This is extremely distracting and reducing user productivity.

I installed your plugin long time ago. When inline edit came into life, I started using it. I learned the keystrokes. I know the plugin is there and I know can invoke it at any time. No need to remind it to me 100 times per hour (literally!). Then you changed the keystroke to invoke it. Fine. I learned new keystroke. Yet, every time I want to select 5 lines, you put tooltip jumping around my screen 5 times. Source code, which is usually static, you somehow made it animated, and it causes user eyes to look for the screen changes. It feels Iike I'm paying to have your advertisement jumping over my screen. And it is harmful, because it sometimes blocks cursor movement and I need to work it around. That hurts my productivity even further.

**Comment by jdorfman** [comment]
Thanks @Nopik 

I will make sure the product team sees this in the weekly community feedback report.

**Comment by esafak** [comment]
It would be great if Cody did not interfere with "select to EOL".

**Comment by mdhender** [comment]
I upgraded to v6.0.14 of the Jetbrains IDE plugin and the issue with selection is fixed. There's also an option to disable entirely.

**Comment by jdorfman** [comment]
Thanks for confirming @mdhender 

@Nopik can you confirm. The lastest release should fix the issue: https://sourcegraph.com/blog/cody-jetbrains-6-0-14-release

**Comment by jdorfman** [comment]
@esafak can you screenshot it that way I can send it to the product team.

**Comment by Nopik** [comment]
I've upgraded to 6.0.14 and basically immediately turned off the whole UI hints and never looked back. Minute ago I enabled it for a moment, and indeed, the tooltip was moved 1 line lower, so it doesn't interact with the cursor, so it is less 'aggressive', definitely. Still, as long as it appears immediately and jump around every time I move cursor, it is a no-go for me and the UI hints will get disabled.
